### PR TITLE
FInd and suggest fixes for geq/leq with int literals

### LINKED ISSUE #38: Support for i <= n
The pattern is not common, but what about detecting

```go
for i := 0 ; i <= 9 ; i++ {

}
```

and suggest fixing it with

```go
for i := range 10 {

}
```

Here are some examples

https://github.com/search?q=%22for+i+%3A%3D+0+%3B+i+%3C%3D+5+%3B+i%2B%2B+%7B%22+language%3Ago&type=code
https://github.com/search?q=%22for+i+%3A%3D+0+%3B+i+%3C%3D+10+%3B+i%2B%2B+%7B%22+language%3Ago&type=code

### PR DESCRIPTION
**Pull request by ckaznocha**
closes #38

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -68,7 +68,6 @@ linters:
     - gofmt
     - gofumpt
     - goimports
-    - gomnd
```
Comment (ccoVeille) [inline .golangci.yml:71]: Why this change in this commit?

gomnd was renamed mnd

https://golangci-lint.run/product/changelog/#v1620

Comment (ckaznocha) [reply .golangci.yml:71]: It was as causing CI to fail and I find mnd to be a lot of noise for not much value so I'm fine without it.

I probably should have removed it in its own PR but I wasn't up for it after adding all the new tests

Comment (ccoVeille) [reply .golangci.yml:71]: Another PR quickly merged would have been better.

You can still update the PR description to explain your moves

Comment (ckaznocha) [reply .golangci.yml:71]: updated
